arm-software/embedded/samples/Makefile.conf

```diff
@@ -13,5 +13,6 @@ MICROBIT_TARGET=--target=armv6m-none-eabi -march=armv6m -mfpu=none -mfloat-abi=s
 AARCH64_TARGET=--target=aarch64-none-elf
 
 CRT_SEMIHOST=-nostartfiles -lcrt0-semihost -lsemihost
+CRT_SEMIHOST_LLVM=-nostartfiles -lcrt0 -lsemihost
```

Comment:
Some ideas how to avoid this difference:
* picolibc provides `crt0.o` if llvm-libc does so, then it would not need `-nostartfiles`
* If we expect that LLVM libc will need `crt0-semihost` in the future, then we can create it as a placeholder right now even if it only provides the same implementation.